README.md

```diff
@@ -1,3 +1,5 @@
+![](docs/source/images/logo.png)
```

Comment:
Nice logo. Personally, I think it's a little large for the readme page ;-)

Reply:
Made the logo smaller, ideally we would have one without background, but let's save that for another PR ;)